src/hasReachedStyle.js

```diff
@@ -1,4 +1,6 @@
-export default function hasReachedStyle(currentStyle, style) {
+/* @flow */
+export default function hasReachedStyle(currentStyle: Object,
```

Comment:
I’ve had mixed experiences with type aliases for simple types in Scala. I guess it would not change much whether we define a new Options type in this case. Do you know whether we can export type definitions from a module like any other variable?